config.py

```diff
@@ -511,19 +529,23 @@ def build_config(
             ),
             "recoil_corrections_file": EraModifier(
                 {
-                    "2016": "data/recoil_corrections/Type1_PuppiMET_2016.root",
+                    "2016preVFP": "data/recoil_corrections/Type1_PuppiMET_2016.root",  # These are likely from Legacy data sets, therefore no difference in pre and postVFP
+                    "2016postVFP": "data/recoil_corrections/Type1_PuppiMET_2016.root",  # These are likely from Legacy data sets, therefore no difference in pre and postVFP
                     "2017": "data/recoil_corrections/Type1_PuppiMET_2017.root",
                     "2018": "data/recoil_corrections/Type1_PuppiMET_2018.root",
                 }
             ),
             "recoil_systematics_file": EraModifier(
                 {
-                    "2016": "data/recoil_corrections/PuppiMETSys_2016.root",
+                    "2016preVFP": "data/recoil_corrections/PuppiMETSys_2016.root",  # These are likely from Legacy data sets, therefore no difference in pre and postVFP
+                    "2016postVFP": "data/recoil_corrections/PuppiMETSys_2016.root",  # These are likely from Legacy data sets, therefore no difference in pre and postVFP
                     "2017": "data/recoil_corrections/PuppiMETSys_2017.root",
                     "2018": "data/recoil_corrections/PuppiMETSys_2018.root",
                 }
             ),
-            "applyRecoilCorrections": SampleModifier({"wjets": True}, default=False),
+            "applyRecoilCorrections": SampleModifier(
+                {"wjets": True}, default=False
+            ),  # ToDo: Shouldn't this be also applied to dy and higgs events?
```

Comment:
good question :D

Reply:
At least the pipelines have been used like this in the past:
https://github.com/KIT-CMS/KITHiggsToTauTau/blob/reduced_trigger_objects/python/data/ArtusConfigs/Run2LegacyAnalysis/METrecoil_shifts.py#L39

Reply:
then, lets add it

Reply:
Added.